mappings/net/minecraft/client/util/ScreenshotUtils.mapping

```diff
@@ -9,7 +9,7 @@ CLASS net/minecraft/class_318 net/minecraft/client/util/ScreenshotUtils
 		ARG 4 messageReceiver
 	METHOD method_1660 getScreenshotFilename (Ljava/io/File;)Ljava/io/File;
 		ARG 0 directory
-	METHOD method_1662 saveScreenshot (Ljava/io/File;Ljava/lang/String;IILnet/minecraft/class_276;Ljava/util/function/Consumer;)V
+	METHOD method_1662 saveScreenshotInner (Ljava/io/File;Ljava/lang/String;IILnet/minecraft/class_276;Ljava/util/function/Consumer;)V
```

Comment:
What does this do differently from `saveScreenshot`? Maybe a more precise name would be better?

Reply:
just like the bindTextureInner change in #870